proxy/core/src/main/resources/cloudstate-common.conf

```diff
@@ -50,6 +50,9 @@ akka {
   cluster {
     shutdown-after-unsuccessful-join-seed-nodes = 60s
 
+    downing-provider-class = "com.lightbend.akka.sbr.SplitBrainResolverProvider"
```

Comment:
Tests are failing. This should be `akka.cluster.sbr.SplitBrainResolverProvider`.